packages/components/src/call-out/types.ts

```diff
@@ -0,0 +1,33 @@
+import type { Icon } from '@wordpress/components';
+import type { ComponentProps, ReactNode } from 'react';
+
+export interface CallOutProps {
+	/**
+	 * A concise headline that clearly communicates the main message or feature being highlighted.
+	 */
+	title: string;
+	/**
+	 * A short paragraph providing supporting context or details to reinforce the title.
+	 */
+	children?: ReactNode;
```

Comment:
I originally had separate props for `description`, `actions`, etc. This works for the example used in the `CallOut` design spec.

However the dashboard mockups use two paragraphs.

![CleanShot 2025-05-14 at 22 36 05@2x](https://github.com/user-attachments/assets/108aed99-d72d-4180-b37c-6602f20fff48)

So that means there are two `<Text>` elements.

Since all elements on the left are vertically stacked, with the same `gap` between each, does it make more sense to just take an arbitrary `children` prop, and let the consumer use `<Text>` and `<Button>` elements as they please? The `VStack` makes sure they'll be arranged correctly.

Reply:
I think using `children` this way seems like a reasonable choice, especially with wanting to reinforce a consistent vertical gap.

A couple alternatives that came to mind, but don't seem like great fits either:

- `description` prop and expecting the consumer to render their own `HStack` if needing multiple paragraphs. Hard to enforce consistency.
- `description` accepting an array of either `<Text>` elements or strings. Arrays of elements aren't very ergonomic since they require `key`s, and arrays of strings might not support the type of flexibility we want in content (links, emphasis, etc.)

Reply:
Why not have `description` with `ReactNode`? The consumer could still pass multiple items with `<>` and get the spacing from the internal VStack, and it's a bit more specific to where the content goes IMO.

Reply:
> Why not have `description` with `ReactNode`?

IMO that basically is a `children` prop, except with a `children` you don't need to wrap in `<>` when you've got multiple.

Perhaps an explicit prop name is better for enforcing consistent usage of the component though?

Reply:
I think to @ntsekouras 's point, when using `children` it should be obvious where those children would be rendered, and I'm not sure it's inherently obvious that it'd be rendered below the heading and above the actions.

Although I think it's the obvious place if you think about it for more than a few seconds, just not _immediately_ obvious. So a soft opinion from me on `children` vs. `description`, anyways.

Reply:
> it should be obvious where those children would be rendered

I think that's a fair point. I'll try adding two separate `description` and `actions` for the sake of making the props explanatory.